torchbox/torchbox.com
# Fix alt text display in image block template and add appropriate help text

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1362070728/posts/156382640)

### Description of Changes Made

This is a small fix to ensure that the alt text display logic on image blocks works as expected:

> 1. if No alt text is specified, use image titile as alt text
> 2. if alt text is specified, use it, instead of the image title
> 3. if the **image is decorative** checkbox is ticked, set alt text to nothing

Additionally, some help text is added to the alt text field, as discussed during the Sprint 1 demo earlier today.

### How to Test

- When you add an image block to a standard page or any other page model, test the three conditions outlined above.
- Check that the help text added serves the intended purpose of ensuring clarity for content editors and authors

### Screenshots

<details>
  <summary>Case 1: no alt text specified</summary>

![image](https://github.com/torchbox/torchbox.com/assets/7713776/dc652287-c0ba-4bf7-a865-f58c653fe1bb)

</details>

<details>
  <summary>Case 2: alt text specified</summary>

![image](https://github.com/torchbox/torchbox.com/assets/7713776/65c77a4e-969c-42c5-8094-f28a33c7f033)


</details>

<details>
  <summary>Case 3: image is decorative</summary>

![image](https://github.com/torchbox/torchbox.com/assets/7713776/4763d858-a554-4e3d-a41c-c43b993eea38)


</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on Linux
  - Latest version of Firefox on Linux

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
